ansible_builder/main.py

```diff
@@ -39,7 +39,13 @@ def __init__(self,
                  container_runtime=constants.default_container_runtime,
                  output_filename=None,
                  no_cache=False,
-                 verbosity=constants.default_verbosity):
+                 verbosity=constants.default_verbosity,
+                 disable_gpg_verify=False,
```

Comment:
This is a bit odd that the default does not match the CLI option default of `True`. Same in `Containerfile` init.

Reply:
Actually a good point, thank you